content/_data/roadmap/roadmap.yml

```
@@ -92,18 +120,29 @@ categories:
         It complements Jenkins Configuration-as-Code stories by preventing undesired manual modifications on running instances.
       status: current
       link: https://github.com/jenkinsci/jep/tree/master/jep/224
+    - name: "Windows Services: YAML Configuration Support"
+      status: current
+      description: >
+        Enhance Jenkins master and agent service management on Windows by
+        offering new configuration file formats and improving settings validation.
+      link: https://www.jenkins.io/projects/gsoc/2020/projects/winsw-yaml-configs/
     - name: Built-in plugin management as-code
       description: >
         Evolution of plugin management capabilities in the Jenkins core and Docker images.
-        It includes adoption of the new Plugin Management tool in distributions, and support of advanced plugin definition formats like YAML.
+        It includes adoption of the new Plugin Management Tool in distributions, and support of advanced plugin definition formats like YAML.
       status: near-term
-      link: https://jenkins.io/sigs/platform/#plugin-management
+      link: https://www.jenkins.io/projects/gsoc/2020/project-ideas/plugin-installation-manager-tool/
     - name: "JCasC: Pluggable configuration sources"
       description: >
         Support external configuration sources in the Jenkins Configuration-as-Code plugin.
         Examples of potential configuration sources: Git, S3 Buckets, Kubernetes CRD.
       status: future
       link: https://github.com/jenkinsci/configuration-as-code-plugin/issues/1365
+    - name: "Better Remoting Monitoring"
+      status: future
+      description: >
+        Support monitoring of Jenkins networking (master to agent communications, etc.) with open source monitoring tools such as Prometheus, Grafana, etc.
```

Possibly safe to use a more direct verb?
```suggestion
        Monitor Jenkins networking (master to agent communications, etc.) with open source monitoring tools such as Prometheus, Grafana, etc.
```

---

I would prefer the original option. We provide a feature (support for something), we do not plan to monitor anything.